lib/services/watcher.go

```diff
@@ -1669,10 +1670,10 @@ func (*accessRequestCollector) notifyStale() {}
 
 // SAMLIdPServiceProviderWatcherConfig is a SAMLIdPServiceProviderWatcher configuration.
 type SAMLIdPServiceProviderWatcherConfig struct {
-	// ResourceWatcherConfig is the resource watcher configuration.
-	ResourceWatcherConfig
-	// SAMLIdPServiceProviders is responsible for fetching access SAML IdP service providers.
-	SAMLIdPServiceProviders
+	// RWCfg is the resource watcher configuration.
+	RWCfg ResourceWatcherConfig
```

Comment:
nit: Make the field private?

Reply:
So believe it or not, the `watcher_test.go` is in a different package:

```
package services_test
```

This isn't easily changeable because importing `github.com/gravitational/teleport/lib/services/local` creates an import cycle. So in order to verify these in the tests, they need to be public.